### DISCUSSION TITLE
Can agent report UTC+2 for Nerthelands?

### DISCUSSION THREAD

**Discussion by hulinning2** [mtconnect/cppagent#508] [Q&A]
I have customer in Netherlands, and they report agent report UTC+1 only although Windows reports UTC+2.
Is this an issue? 
Thanks

**Comment by wsobel** [comment]
The agent should only report UTC timestamps. I don’t know why it would report UTC + 1 or any specific time zone except UTC. 

Is there more information you can send me?

Thanks,
- W

**Comment by hulinning2** [comment]
Will,
I checked online and I think they are mentioning about Daylight saving time which becomes UTC+2 as shown in the link:
[https://www.worlddata.info/europe/netherlands/timezones.php#google_vignette](https://www.worlddata.info/europe/netherlands/timezones.php#google_vignette)

**Comment by wsobel** [comment]
This should not be an issue since UTC does not change for daylight savings. I’m concerned that the agent would be emitting anything other than UTC. 

Best
W

**Comment by hulinning2** [comment]
Does our UTC change during Daylight saving time in US?
I think they expect to see UTC+2 due to Daylight saving time? Is that correct assumption or not?

Thanks

**Reply by wsobel** [reply]
UTC never changes. Local time zones change. They need to convert from UTC to their time zone. That may be the confusion. It is also why the time should always be followed by a Z to indicate it is in UTC.
